utils/token.js

```diff
@@ -1,21 +1,21 @@
-const jwt = require('jsonwebtoken');
-const config = require('../config/auth');
+const jwt = require("jsonwebtoken");
+const config = require("../config/auth");
 
 // Generate an Access Token for the given User ID
 function generateAccessToken(userId) {
-	const expiresIn = '1 hour';
-	const { audience, issuer, secret } = config.authentication.token;
-	
-	const token = jwt.sign({}, secret, {
-		expiresIn,
-		audience,
-		issuer,
-		subject: userId.toString(),
-	});
+  const expiresIn = "1 hour";
```

Comment:
we should assign tick instead of string

Reply:
I have not gone through this method it was written previously by you so i have used it.
infact these kind of token cannot be part of the application they have to be saved in central persistent storage like redis where ttl of it is managed and expiry time is not attached explicitly for the token